pywwt/nbextension/static/interactive_figure/interactive_figure.js

```diff
@@ -0,0 +1,288 @@
+var wwtIntialState;
+
+var wwt;
+var wwt_ready = 0;
```

Comment:
Inconsistent naming with `wwt_ready`; I think it's good to aim to consistently use `lowerCamelCase` for JS identifiers, in which case we'd have `wwtReady`.

Reply:
I ended up removing `wwt_ready` because it’s not useful for the interactive figure. (It was used to signal back to pywwt that the widget was ready).